packages/core/src/external/carequality/ihe-gateway-v2/outbound/ihe-gateway-v2-logic.ts

```diff
@@ -151,16 +162,41 @@ export async function createSignSendProcessXCPDRequest({
       cxId,
       index,
     });
+    if (result.patientMatch) {
+      try {
+        // TODO not sure if we should retry on timeout
+        await executeWithNetworkRetries(async () => axios.post(pdResponseUrl, result), {
+          initialDelay: 100,
+          maxAttempts: 5,
+          httpStatusCodesToRetry: [502, 504],
+          log,
+        });
+      } catch (error) {
+        const msg = "Failed to send PD response to internal CQ endpoint";
+        const extra = { cxId, patientId, result };
+        log(`${msg} - ${errorToString(error)} - ${JSON.stringify(extra)}`);
+        capture.error(msg, { extra: { ...extra, error } });
+      }
+    }
     try {
-      // TODO not sure if we should retry on timeout
-      await executeWithNetworkRetries(async () => axios.post(pdResponseUrl, result), {
-        initialDelay: 100,
-        maxAttempts: 5,
-        httpStatusCodesToRetry: [502, 504],
-        log,
+      const filePath = createHivePartitionFilePath({
+        cxId,
+        patientId,
+        keys: {
+          stage: "pd",
+          requestId: result.id,
+          gatewayOid: result.gateway.oid,
```

Comment:
I have confirmed with @jonahkaye that this segments each response into its own path.